app/assets/javascripts/preferences/panes/listed/BlogItem.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from '@/components/Button';
+import { HorizontalSeparator } from '@/components/shared/HorizontalSeparator';
+import { LinkButton, Subtitle } from '@/preferences/components';
+import { WebApplication } from '@/ui_models/application';
+import { Action, SNComponent, SNItem } from '@standardnotes/snjs/dist/@types';
+import { JSXInternal } from 'preact/src/jsx';
+import React, { useState } from 'react';
+
+type Props = {
+  item: SNComponent;
+  showSeparator: boolean;
+  disabled: boolean;
+  disconnect: (item: SNItem) => void;
+  application: WebApplication;
+};
+
+export const BlogItem = ({
+  item,
+  showSeparator,
+  disabled,
+  disconnect,
+  application,
+}: Props): JSXInternal.Element => {
```

Comment:
Instead of specifying the output type, it's better to specify that BlogItem is a FunctionComponent, like so:
`export const BlogItem: FunctionComponent<Props> = ...`
This way, the `item` property's type will be inferred and all the `(item as any)` casts can be removed.

Reply:
Removing the type casts gives me this error: `Property 'actions' does not exist on type 'ComponentPackageInfo'.`